### DISCUSSION TITLE
A DLNA ouput module would be incredible

### DISCUSSION THREAD

**Discussion by Futureknows** [audacious-media-player/audacious#112] [Ideas]
I have a DLNA output from Fedora to my receiver, but Fedora piperwire resamples HD audio down to 16/44. If Audacious could directly output to the DLNA rather than pipewire, it could be a full uncompressed 24/192 streaming solution. The received handles 24/192 FLACs streamed from Synology perfectly.

**Comment by radioactiveman** [comment]
PipeWire can be configured to use specific sample rates to avoid resampling to 48 kHz.
Audacious until version 4.3 did not handle this, but 4.4 does if PipeWire is configured accordingly and the audio file has the same bit rate.

I hope this helps. :)

See also:
- https://github.com/orgs/audacious-media-player/discussions/96
- https://wiki.archlinux.org/title/PipeWire#Changing_the_allowed_sample_rate(s)
